blocks/extensions/__init__.py

```diff
@@ -128,6 +128,24 @@ def on_interrupt(self):
         pass
 
 
+class CallbackName(str):
```

Comment:
This is a question of terminology I guess. For me a callback is a function which is called, not the string which is passed to `dispatch`. That is `def before_training, def after_training` and so on. That's why I was confused. Seems like you call the string message passed to `dispatch` a callback, right?